specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/storage.json

```diff
@@ -2574,6 +2574,19 @@
     },
     "StorageAccountPropertiesCreateParameters": {
       "properties": {
+        "publicNetworkAccess": {
```

Comment:
What's "none" means? Can customer set "None" from rest API?

Reply:
Below is the expected behavior for the above property.
1. Enabled & None = Allow client connection requests.
2. Disabled = Block connections over public endpoints even if they match the IP/VNet firewall rules.

Will share the sample rest APIs for this offline.

Reply:
Since "None is same as "Enabled", Why we need "None"? Is it the default value?
Can customer set "None" from rest API?  Or none is readonly?

From many other properties only have enabled/disable status, we use a bool value type instead of enum.
The .net SDK will have it as "bool?". The default value is server not return it, so .net SDK will have it as null. 
You can reference https://github.com/Azure/azure-rest-api-specs/blob/3893616381e816729ef9cdd768e87fb2845e189d/specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/storage.json#L3044

Reply:
1. Yes, "None" is equal to the default value (null in case the property is never set)
2. Customer can set the property to "None" from rest API. It is not readonly.
3. The requirement for the property stated that the values should be strings and not bool which is why we have used the string values. In the future we may allow additional values for this property and in those cases a bool would not be sufficient.

Reply:
@raomanus 
Please indicate how comes "The requirement for the property stated that the values should be strings and not bool "
It still looks odd that has 2 option Enabled & None with same meaning.

Besides that, please answer: can we set "None" when create/update account? If so, is there any difference than set "enabled" in create/update account?